File: GradeReceiver.php

<?php
    $studentID1 = $_GET["studentID1"];
    $grade1 = $_GET["grade1"];
    $studentID2 = $_GET["studentID2"];
    $grade2 = $_GET["grade2"];
    $studentID3 = $_GET["studentID3"];
    $grade3 = $_GET["grade3"];

    echo "Student 1: $studentID1  Grade: $grade1";
    echo "Student 2: $studentID2  Grade: $grade2";
    echo "Student 3: $studentID3  Grade: $grade3";

    $sum = $grade1 + $grade2 + $grade3;
    $average = $sum/3;

    echo "The cumulative average of the grades is $average";
?>

File: GradeReceiver.py
from flask import Flask, request, render_template

app = Flask(__name__)
@app.route('/')
def page():
    return render_template('GradeStudentIDInfo.html')

@app.route('/GradeReceiver', methods=['GET', 'POST'])
def GradeReceiver():
    if request.method == 'GET':
        student1 = request.form['studentID1']
        student2 = request.form['studentID2']
        student3 = request.form['studentID3']
        grade1 = request.form['grade1']
        grade2 = request.form['grade2']
        grade3 = request.form['grade3']
        average = ((int(grade1) + int(grade2) + int(grade3)) / 3)
        output = "Student ID: %s      Grade: %s" \
                 "Student ID: %s      Grade: %s" \
                 "Student ID: %s      Grade: %s" \
                 "The cumulative average grade is %s"

        return output % (int(student1), int(grade1), int(student2), int(grade2), int(student3),
                         int(grade3), int(average))

if __name__ == '__main__':
    app.run(debug=True)


